src/monitor/persistence/hardware_cpu_info.rs

```diff
@@ -2,12 +2,12 @@ use sqlx::SqliteConnection;
 
 use crate::monitor::models::get_hardware_info::HardwareCpuInfo;
 
-use super::{get_sql_connection, FetchId, SQLITE_DB_CONN_STR};
+use super::{get_default_sql_connection, FetchId};
 
 const HARDWARE_CPU_INFOS_TABLE_NAME: &str = "cpu_infos";
 
 pub(super) async fn insert_hardware_cpu_info(info: &HardwareCpuInfo) -> Result<(), sqlx::Error> {
-    let mut conn = get_sql_connection(SQLITE_DB_CONN_STR).await?;
+    let mut conn = get_default_sql_connection().await?;
```

Comment:
felt no need to let each module get a connection string and passing it to the function, as far as the vision of the project goes the db connection config is going to be from a single place

Reply:
since we have single db file, yeah that was unnecessary.

Reply:
even in the future if we add a feature to connect to 3rd party db sources, having the connection string in a single place will still be enough